grammar.js

```diff
@@ -4,19 +4,18 @@ const PHP = require('tree-sitter-php/grammar');
 // TODO array return types https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html#arrays
 // PHPDoc tags: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/index.html#tag-reference
 // PHPDoc docs appear to use these conventions
+// TODO inline tags in inline tags (allowed for @internal)
 //   @verbatimElement [required element] [<optional element>]
 
 module.exports = grammar({
   name: 'phpdoc',
 
   extras: $ => [
-    token(
-      seq(/(\r)|(\r\n)|(\n)/, /[ \t]*/, repeat('*'), /[ \t]*/)
```

Comment:
@claytonrcarter Could you explain what was the idea here? It seems you want to deal with MacOs/Windows linebreaks here. But I've never seen this in any other parser. So I wonder what is different in the phpdoc case.

Reply:
`extras` appears to be unchanged since the first commit, before I was even involved. I'm totally fine to alter/adjust them however we like, so long as we have tests to validate the changes. (Which we do.)

Reply:
Cool. I also prefer to keep things simple, so let's keep the reverted changes then.